contracts/TokenMigration.aes

```diff
@@ -14,46 +19,50 @@ payable contract TokenMigration =
           migrations_count = count,
           map_transfered_accounts = {} }
```

Comment:
For N ethereum AE holders, after all people migrate their tokens to AE the storage complexity for this contract is O(N) - each node in the network will permanently store this mapping which will have a similar storage footprint as just storing all the ETH accounts with the balances in the smart contract. Unfortunately the nodes will never be able to forget this mapping which will slow down light/warp like syncs.

If on the other hand we just store a mapping from the ETH address to the AE tokens to be claimed, with time we can decrease the storage footprint instead of increasing it :) Additionally processing claims will take less gas.

As we need to hardcode special deployment logic in the node anyway why can't we just take a json file with the mapping and then just create the contract store manually in the deployment logic? No heavy create_contract_tx or large bytecode sizes would be required :) The only downside is that the nodes would need to store the mapping in a json file - but we have done exactly this for the previous hard forks :) so I think we might do it one more time :)